bundles/binding/org.openhab.binding.zwave/database/fibaro/fgms001.xml

```diff
@@ -67,9 +67,11 @@ Formula to calculate the number of pulses: pulses = (value + 1)]]>
 			<Maximum>3</Maximum>
 			<Size>1</Size>
 			<Label lang="en">PIR Sensor Window Time</Label>
-			<Help lang="en"><![CDATA[Sets the number of moves required for the PIR sensor to report motion. The lower the value, the less sensitive the PIR sensor. It’s not recommended to modify this parameter settings.
+			<Help lang="en"><![CDATA[Period of time during which the number of moves set in parameter 3 
```

Comment:
Can I suggest adding some formatting - since this is already using the CDATA block, you can add some BReaks into the HTML to better format it - otherwise all this will be shown as a single (wrapped) line.